cmd/utils/flags.go

```diff
@@ -765,6 +765,12 @@ var (
 		Usage:    "Disables the peer discovery mechanism (manual peer addition)",
 		Category: flags.NetworkingCategory,
 	}
+	DiscoveryV4Flag = &cli.BoolFlag{
+		Name:     "v4disc",
```

Comment:
I like `discv4` and `discv5` better as flags. Thoughts on marking `v5disc` deprecated and add a new one with the same functionality where the version is a postfix?

Reply:
why not!

Reply:
done in 9ec6f5bc11115dcdd2198d05cae1fefc85e6bc65